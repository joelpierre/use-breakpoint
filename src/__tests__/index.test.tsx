import { renderHook } from '@testing-library/react';
import { useBreakPoint } from '..';

describe('useBeakPoint', () => {
  it('should return min and max breakpoints', () => {
    const { result } = renderHook(() => useBreakPoint());
    expect(result.current).toMatchSnapshot();
  });

  it('should return min breakpoints', () => {
    const { result } = renderHook(() => useBreakPoint('min'));
    expect(result.current).toMatchSnapshot();
  });

  it('should return max breakpoints', () => {
    const { result } = renderHook(() => useBreakPoint('max'));
    expect(result.current).toMatchSnapshot();
  });
});
