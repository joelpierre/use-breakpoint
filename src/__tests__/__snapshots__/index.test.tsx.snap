// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`useBeakPoint > should return max breakpoints 1`] = `
{
  "is2Xl": false,
  "isLg": false,
  "isMd": false,
  "isSm": false,
  "isXl": false,
  "isXs": false,
}
`;

exports[`useBeakPoint > should return min and max breakpoints 1`] = `
{
  "max": {
    "is2Xl": false,
    "isLg": false,
    "isMd": false,
    "isSm": false,
    "isXl": false,
    "isXs": false,
  },
  "min": {
    "is2Xl": false,
    "isLg": false,
    "isMd": false,
    "isSm": false,
    "isXl": false,
    "isXs": false,
  },
}
`;

exports[`useBeakPoint > should return min breakpoints 1`] = `
{
  "is2Xl": false,
  "isLg": false,
  "isMd": false,
  "isSm": false,
  "isXl": false,
  "isXs": false,
}
`;
